<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window</class>
 <widget class="QMainWindow" name="Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>954</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Window</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>826</width>
         <height>546</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>954</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea|Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTreeWidget" name="treeWidget">
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actAddFolder"/>
   <addaction name="actAddFiles"/>
   <addaction name="actDeleteItem"/>
   <addaction name="actScanItems"/>
   <addaction name="actZoomOut"/>
   <addaction name="actZoomIn"/>
   <addaction name="actZoomRealSize"/>
   <addaction name="actZoomFitW"/>
   <addaction name="actZoomFitH"/>
   <addaction name="actDockFloat"/>
   <addaction name="actDockVisible"/>
   <addaction name="actQuit"/>
  </widget>
  <action name="actAddFolder">
   <property name="text">
    <string>AddFolder</string>
   </property>
  </action>
  <action name="actAddFiles">
   <property name="text">
    <string>AddFiles</string>
   </property>
  </action>
  <action name="actZoomIn">
   <property name="text">
    <string>ZoomIn</string>
   </property>
  </action>
  <action name="actZoomOut">
   <property name="text">
    <string>ZoomOut</string>
   </property>
  </action>
  <action name="actZoomRealSize">
   <property name="text">
    <string>ZoomRealSize</string>
   </property>
  </action>
  <action name="actZoomFitW">
   <property name="text">
    <string>ZoomFitW</string>
   </property>
  </action>
  <action name="actDeleteItem">
   <property name="text">
    <string>DeleteItem</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actZoomFitH">
   <property name="text">
    <string>ZoomFitH</string>
   </property>
  </action>
  <action name="actScanItems">
   <property name="text">
    <string>ScanItems</string>
   </property>
  </action>
  <action name="actDockVisible">
   <property name="text">
    <string>DockVisible</string>
   </property>
  </action>
  <action name="actDockFloat">
   <property name="text">
    <string>DockFloat</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actQuit</sender>
   <signal>triggered()</signal>
   <receiver>Window</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>476</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
